shmounts/setup-shmounts.c

```diff
@@ -66,6 +66,12 @@ int mkdir_p(const char *dir, mode_t mode)
 	return 0;
 }
 
+__attribute__((noreturn))
+static void die(const char *s)  {
+	perror(s);
```

Comment:
This will display the value of errno. This is fine but may be non-obvious during usage.